bluebrain/repo-bluebrain/packages/py-visual-cortex-analysis/package.py

```diff
@@ -12,7 +12,7 @@ class PyVisualCortexAnalysis(PythonPackage):
     homepage = "https://bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis"
     git = "ssh://git@bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis.git"
 
-    version("0.1.0.dev0", tag="visual-cortex-analysis-v0.1.0.dev0")
+    version("0.2.1.dev2", tag="visual-cortex-analysis-v0.2.2.dev1")
```

Comment:
Please match the version and tag

Reply:
Don't use `dev` versions, either.

Reply:
I should make a full version release?

Reply:
@matz-e  this request is blocking the merge. I think you need to  indicate that it has been addressed.